packages/flutter_test/lib/src/controller.dart

```diff
@@ -273,6 +273,17 @@ abstract class WidgetController {
     });
   }
 
+  /// Dispatch a pointer down at the center of the given widget, assuming it is
+  /// exposed.
+  ///
+  /// If the center of the widget is not exposed, this might send events to
+  /// another object.
+  Future<TestGesture> press(Finder finder, { int pointer }) {
```

Comment:
The gesture is still held down after this returns right? Perhaps call it pressDown or something to make it not sound like a complete press down+up sequence.